scss/tech.scss

```diff
@@ -15,18 +15,28 @@
   margin: 10px 20px;
   min-height: 280px;
   width: 180px;
-
+  
   &::before {
     bottom: 10px;
     border: 1px solid $green;
     border-radius: 5px;
     content: '';
+    height: 100%;
```

Comment:
you need to either set `width: 100%`, `height: 100%` and remove the `bottom` and `right` properties or remove this and make a transition in the left top right and bottom properties by redefining de 4 values